Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -2041,13 +2034,21 @@ extension BrowserViewController {
         // Ex. Internet connection is unstable due to which
         // leanplum isn't loading or taking too much time
         DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
-            guard self.onboardingUserResearch?.updatedLPVariables != nil else {
+            guard LeanPlumClient.shared.onStartLPVariable != nil else {
                 return
             }
-            Sentry.shared.send(message: "Onboarding Research: LP State - \(LeanPlumClient.shared.lpState.rawValue)", tag: .leanplum, severity: .debug, description: "Leanplum server too slow")
-            Sentry.shared.send(message: "Failed to fetch A/B test variables from LP")
+            let lpStartStatus = LeanPlumClient.shared.onStartResponseStatus
+            var lpVariableValue = true
+            // Condition: LP has already started but we missed onStartLPVariable callback
+            if lpStartStatus, let boolValue = LPVariables.showOnboardingScreenAB?.boolValue() {
+                lpVariableValue = boolValue
```

Comment:
This part is easy to refactor but I kept it this way so that I could easily send sentry event(s) or put prints etc. otherwise assigning .versionV1 is fairly straight forward. 

P.S. all this code will be removed and move to onboarding user research (have something in mind) just need to know why LP is doing what its doing.